src/main/java/ro/msg/learning/shop/controller/OrderController.java

```diff
@@ -0,0 +1,25 @@
+package ro.msg.learning.shop.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.msg.learning.shop.dto.OrderDto;
+import ro.msg.learning.shop.model.Order;
+import ro.msg.learning.shop.service.OrderService;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/orders")
+public class OrderController {
+
+    private final OrderService orderService;
+
+    @PostMapping
+    public ResponseEntity<Order> createOrder(@RequestBody OrderDto orderDto) {
```

Comment:
You should never expose an entity class directly out of a controller. You should use dtos to expose data outside your application.